packages/config-array/src/config-array.js

```diff
@@ -89,6 +90,9 @@ const CONFIG_WITH_STATUS_UNCONFIGURED = Object.freeze({
 	status: "unconfigured",
 });
 
+// Match two leading dots followed by a slash or the end of input.
+const EXTERNAL_PATH_REGEX = /^\.\.(\/|$)/u;
```

Comment:
```suggestion
const EXTERNAL_PATH_REGEX = /^\.\.(?:\/|$)/u;
```

Since we're not using the group.